flexible/java-8/sparkjava/jenkins.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Fail on non-zero return and print command to stdout
+set -xe
```

Comment:
Is this file needed?

Reply:
I have just copied over the existing projects from the flexible/ folder to flexible/java-8 and this file was already there as a part of https://github.com/GoogleCloudPlatform/java-docs-samples/pull/538. I think it is okay to leave it as is since this repository is for samples. wdyt?